dlp/snippets/noxfile.py

```diff
@@ -0,0 +1,292 @@
+# Copyright 2023 Google LLC
```

Comment:
This file should not be checked in as a global `noxfile.py` is to be used by CI/CD. We need this file only in case of local testing.